apps/api/src/utils/email_handler.py

```diff
@@ -54,15 +65,17 @@ async def send_guest_login_email(email: EmailStr, passphrase: str) -> None:
 
 
 async def send_decision_email(
-    applicant_batch: Iterable[tuple[str, EmailStr]], decision: Decision
+    applicant_batch: Iterable[tuple[str, EmailStr]],
+    decision: Decision,
+    application_type: Role,
```

Comment:
Suggestion: might help to constrain this type to require one of the three roles as a Literal